### PR TITLE
TDL-13904: Upgraded event stream schema.

### PR DESCRIPTION
**Pull request by dbshah1212**
# Description of change
Removed duplication of other stream's schema in Event stream schema.

# Manual QA steps
 - Run discover mode and verified that a catalog doesn't change.
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

File: tap_chargebee/schemas/item_model/events.json [hunk 1]
```diff
@@ -0,0 +1,571 @@
...
+          "items": {
+            "type": ["null","object"],
+            "properties": {
+              "id": {
+                "type": ["null", "string"],
+                "maxLength": 50
+              },
+              "customer_id": {
+                "type": ["null", "string"],
+                "maxLength": 50
```
Comment (karanpanchal-crest) [inline tap_chargebee/schemas/item_model/events.json:86]: Remove 'maxLength' or 'minLength' properties

Comment (dbshah1212) [reply tap_chargebee/schemas/item_model/events.json]: Removed

File: tap_chargebee/streams/base.py [hunk 2]
```diff
@@ -26,6 +26,36 @@ def write_schema(self):
             key_properties=self.KEY_PROPERTIES,
             bookmark_properties=self.BOOKMARK_PROPERTIES)
 
+    def get_abs_path(self, path):
+        return os.path.join(os.path.dirname(os.path.realpath(__file__)), path)
+
+    def load_shared_schema_refs(self):
```
Comment (karanpanchal-crest) [inline tap_chargebee/streams/base.py:32]: Add comments in the functions for better understanding

Comment (dbshah1212) [reply tap_chargebee/streams/base.py:32]: Added

**Review by karanpanchal-crest — APPROVED** [review]

**Review by KAllan357 — APPROVED** [review]

**Review by KAllan357 — APPROVED** [review]

**Pull request merged**